components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -233,6 +236,25 @@ - (void)setupTextView {
   self.textView.opaque = NO;
 }
 
+// It's possible we've been passed a text view that's not our special class. We can handle that.
+// It's also something we guarantee to always be there. So, if it's nil, we make a new one.
+- (void)textViewSafetyCheck {
```

Comment:
Naming is hard.
Consider: 
- (void)promoteToMDCIntrinsicTextView

Reply:
It's hard because it also makes sure that it isn't nil.